draft-ietf-tls-tls13.md

```diff
@@ -4023,32 +4030,35 @@ and renegotiation_info {{RFC5746}}.
 
 ## Cipher Suites
 
-A symmetric cipher suite defines the pair of the AEAD cipher and hash
-function to be used with HKDF.
+A symmetric cipher suite defines the pair of the AEAD algorithm and hash
+algorithm to be used with HKDF.
 Cipher suite names follow the naming convention:
 
 ~~~
-   CipherSuite TLS_CIPHER_HASH = VALUE;
+   CipherSuite TLS_AEAD_HASH = VALUE;
 ~~~
 
 | Component | Contents |
 |:----------|:---------|
-| TLS     | The string "TLS" |
-| CIPHER    | The symmetric cipher used for record protection |
+| TLS       | The string "TLS" |
+| AEAD      | The AEAD algorithm used for record protection |
```

Comment:
TLS 1.3 is incapable of handling non-AEAD symmetric ciphers (which isn't really a concrete thing). An AEAD is a very particular precomposed thing with a particular interface. TLS 1.3 requires that precomposed thing now, so we should specify in terms of that.

For instance, this sentence does not reflect how any good crypto API works:
> The “CIPHER” component commonly has sub-components used to designate the cipher name, bits, and mode, if applicable. For example, “AES_256_GCM” represents 256-bit AES in the GCM mode of operation.

A good crypto library will not parametrize "give me an AES thingy" with a CBC, CTR, or GCM argument. These three all give back different things with very different properties. This is deciding AEADs and RFC 5116 is a thing.